Editing notes using the github.dev editor.md

```diff
@@ -0,0 +1,47 @@
+---
+aliases: 
+- 
+tags:
+- seedling
+publish: true
+---
+
+# Editing notes using the github.dev editor
+
+%% Add a description below this line. It doesn't need to be long: one or two sentences should be a good start. %%
+
+This guide will cover the basics of using the github.dev editor for the [[Obsidian Hub]]. This editor is great when you are making quick edits that don't need any Obsidian-specific functionality.
+
+## Opening the github.dev editor
+
+To open the editor you'll usually have two options: 
+1. On the Hub's publish site, each note has an "Edit in GitHub" button in it's footer
+2. When you are in the Hub's repository, pressing <kbd>.</kbd> will take you to the editor. 
+
+## Making changes to the notes
+
+1. To make changes, you can select files from the file explorer (1), and make edits to the note (2) as you would on any text editor or paste any changes made in Obsidian to existing notes. To add new files, you can also drag from your system's file explorer to the file explorer in the editor (1).
+	![[github.dev editor.png]]
+2. Once you are ready to submit your changes, switch to the version control tab. You'll see the names of the files you've modified under the "Changes" section:
+    ![[github.dev editor version control.png|300]]
+3. Add the files you want to submit by pressing the <kbd>+</kbd> sign (2), and write a message describing the changes you've made(1): 
+	![[github.dev editor stage changes.png|300]]
+4. To submit your files, you have to press the <kbd>✓</kbd> button at the top. 
+	The first time you make an edit to the hub, you'll get a message explaining that GitHub will create a copy of the repository on your account. This is called a **Fork**.
+	![[github.dev editor commit changes.png]]
+5. Next, it will ask you to give a name to the *branch* where you want to save your changes. You can choose a new name for it, or just press enter. ^8cdfd9
+	![[github.dev editor create branch.png]] 
+6. GitHub will let you know it will switch now to the copy of the repository in your account. And once it reloads you'll see that your changes have been saved.
+	![[github.dev switch to fork.png|300]]
+	![[github.dev fork view.png]]
+
+## Submitting changes to the Hub
+
+Using git terminology, this is called creating a **Pull Request**.
+
+1. Clicking on [this](https://github.com/obsidian-community/obsidian-hub/compare) link is equivalent to going to the Hub repository, clicking on the Pull Requests tab and choosing New. Click on compare across forks, and choose yours on head repository. For the branch, switch to the one you chose on [[#^8cdfd9|step 6]] on the previous section:
+	![[github compare forks.png]]
```

Comment:
The info in the pictures is great. I think we should review them after the page is published, for readability and accessibility.

And if we do re-do the images with smaller windows and large font sizes, maybe we could use files from the Hub, for context?

Here's an example of an existing page, on the Publish site, so show the type of problem:

![image](https://user-images.githubusercontent.com/4840096/150657133-138f1c07-75f5-4711-a073-8c806e0cab64.png)

Reply:
If need be, these images can also be opened at full size with right click, open in new window. As you said below, it is something we can improve later.